src/components/App.css

```diff
@@ -23,39 +36,183 @@ h1 {
 
 li {
   margin: 10px 20px;
+  color: #000;
 }
 
-.book-space,
 .categories-space {
   display: flex;
   justify-content: center;
   align-items: center;
   padding: 100px 0;
 }
 
+.book-lists {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  background-color: #f5f6fa;
+}
+
+.add-new-book {
+  background-color: #f5f6fa;
+  padding: 40px;
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.add-form {
+  width: 90%;
+  border-top: 2px solid #e8e8e8;
+}
+
 .link-tag {
   cursor: pointer;
+  font-weight: 200;
 }
 
 /* STYLES FOR BOOK ITEM */
 
 .book-item {
-  width: 950px;
-  border: 1px solid #000;
+  width: 88%;
+  border: 1px solid#e8e8e8;
+  background: #fff;
   padding: 15px;
   display: flex;
   justify-content: space-between;
+  margin-bottom: 20px;
 }
 
-span {
-  color: rgb(65, 63, 63);
+.second-info {
+  display: flex;
+  justify-content: space-between;
+}
+
+.right-info {
+  padding: 0 50px;
 }
 
 button {
   margin: 10px 0;
-  padding: 2px 8px;
+  padding: 2px 0;
+  cursor: pointer;
 }
 
 .buttons {
   margin-right: 5px;
+  color: #4386bf;
+  border: none;
+  background: none;
+  outline: none;
+  font-weight: 200;
+}
+
+.buttons:first-child {
+  border-right: 2px solid #e8e8e8;
+  padding: 5px;
+}
+
+.buttons:last-child {
+  border-left: 2px solid #e8e8e8;
+  padding: 5px;
+}
+
+.vertical-border {
+  border-right: 2px solid #e8e8e8;
+  padding-right: 40px;
+}
+
+.center-info {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  gap: 20px;
+}
+
```

Comment:
- [ ] As the requirements say that we should follow the template design, Then I think it would be better if you center your progress-bar container. kindly check here the difference :point_down: 

Your design | The template
----|----
![image](https://user-images.githubusercontent.com/65068771/155749486-e2d26483-9084-4974-97eb-4d582c8e3083.png) | ![image](https://user-images.githubusercontent.com/65068771/155749608-bded3003-d6d2-4a5b-b046-9111025b4dc8.png)

Reply:
I have